### PR TITLE
Improve version check

### PR DESCRIPTION
**Pull request by forslund**

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -32,11 +32,21 @@
...
-if sum_of_core >= compatible_core_version_sum:
-    import mycroft.client.enclosure.display_manager as DisplayManager
+compatible_core = True
+
+try:
+    from mycroft.version import check_version
+    if check_version('0.8.19'):
+        import mycroft.client.enclosure.display_manager as DisplayManager
+        compatible_core = True
+except:
```
Comment (MatthewScholefield) [inline __init__.py]:
```Python
except ImportError:
```
Maybe limit to that?

Comment (forslund) [reply __init__.py]: Excellent suggestion, done

File: __init__.py [hunk 2]
```diff
@@ -32,11 +32,21 @@
...
+compatible_core = True
+
+try:
+    from mycroft.version import check_version
+    if check_version('0.8.19'):
+        import mycroft.client.enclosure.display_manager as DisplayManager
+        compatible_core = True
+except ImportError:
+    compatible_core_version_sum = 27
+    sum_of_core += CORE_VERSION_MAJOR
```
Comment (MatthewScholefield) [inline __init__.py]: sum_of_core is not defined

File: __init__.py [hunk 3]
```diff
@@ -32,11 +32,21 @@
 
 LOGGER = getLogger(__name__)
 
-compatible_core_version_sum = 27
-
-sum_of_core = CORE_VERSION_MAJOR + CORE_VERSION_MINOR + CORE_VERSION_BUILD
-if sum_of_core >= compatible_core_version_sum:
-    import mycroft.client.enclosure.display_manager as DisplayManager
+compatible_core = True
```
Comment (MatthewScholefield) [inline __init__.py]: Should be False by default.

**Comment by forslund** [comment]
Oh dear, very sloppy job on my part. feel free to squash when you merge. Destroying all evidence that I screwed up ;)     (multiple times)

**Comment by MatthewScholefield** [comment]
Lol, no prob. Looks good now. I also should've noticed all the errors rather than pointing them out one by one as it failed.

**Pull request merged**